docs/user/installation.rst

```diff
@@ -5,15 +5,6 @@
 Installation
 ============
 
-Preparation
-===========
-
-You need to edit your :code:`PYTHONPATH` to include the garage directory:
-
-.. code-block:: bash
```

Comment:
You should tell folks to `pip install -e .` after `source activate garage` instead, or else they won't have `garage` in their path.

Reply:
Is there no way to do it from within the docker files?

Reply:
I made changes to Dockerfile.ci, in an attempt to automate this instead of asking users to `pip install` 🤞

Reply:
Users don't use Dockerfile.ci--it's currently only used by the CI.

Reply:
The setup scripts were already updated to run `pip install`

[setup_osx](https://github.com/rlworkgroup/garage/pull/321/files#diff-9b3afda2da20b8b001323b10f7fa3baeR228)

[setup_linux](https://github.com/rlworkgroup/garage/pull/321/files#diff-078766c0ef4038e6f79e94cec003a016R210)

Verified manually on macOS 10.13.6